### PR TITLE
Xcode result bundle to checks 1.0.0

### PR DESCRIPTION
**Pull request by kishikawakatsumi**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/xcode-result-bundle-to-checks/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+run_if: .IsCI
+inputs:
+  - opts:
+      description: |
+        Path to the xcresult bundle.
+      is_expand: true
+      is_required: true
+      summary: Path to the xcresult bundle.
+      title: Result bundle path
+    xcresult_path: $BITRISE_XCRESULT_PATH
```
[the remark this answers is not in the document]
Comment (kishikawakatsumi) [reply steps/xcode-result-bundle-to-checks/1.0.0/step.yml]: Thanks. It sounds good.

[the remark this answers is not in the document]
Comment (mateherber) [reply steps/xcode-result-bundle-to-checks/1.0.0/step.yml]: Thank you! My last request, I promise: please reflect these changes in the step repository as well:
https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks/blob/main/step.sh
https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks/blob/main/step.yml

File: steps/xcode-result-bundle-to-checks/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  Generates a human-readable test report from the Xcode result bundle and shows it on GitHub Checks.
+description: |
+  Generates a human-readable test report from the Xcode result bundle and shows it on GitHub Checks.
+website: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks
+source_code_url: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks
+support_url: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks/issues
+published_at: 2021-10-25T19:38:24.734504+09:00
+source:
+  git: https://github.com/kishikawakatsumi/bitrise-step-xcode-result-bundle-to-checks.git
+  commit: 5dfc36f022669c89faf63028cf91b41c005cbfa5
```
Comment (mateherber) [inline steps/xcode-result-bundle-to-checks/1.0.0/step.yml:12]: tag `1.0.0` has to be moved to latest commit (after all changes are done) and hash here has to be updated to that hash. Or alternatively you can close this PR and re-open with all changes applied and the version tag moved to that commit.

File: steps/xcode-result-bundle-to-checks/1.0.0/step.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kishikawakatsumi) [reply steps/xcode-result-bundle-to-checks/1.0.0/step.yml]: Done!

File: steps/xcode-result-bundle-to-checks/1.0.0/step.yml (hunk 2 above)
Comment (kishikawakatsumi) [reply steps/xcode-result-bundle-to-checks/1.0.0/step.yml]: Fixed. Thanks for the review :D

**Comment by mateherber** [comment]
Thank you @kishikawakatsumi for your perseverance during this review process and again for your contribution, merging now!

**Review by mateherber — APPROVED** [review]

**Pull request merged**